optuna_dashboard/_cli.py

```diff
@@ -107,21 +116,49 @@ def main() -> None:
         help="directory to store artifact files",
         default=None,
     )
+    parser.add_argument(
+        "--from-config",
+        help="configuration file in TOML format",
+        type=str,
+        default=None,
+    )
     parser.add_argument("--version", "-v", action="version", version=__version__)
     parser.add_argument("--quiet", "-q", help="quiet", action="store_true")
     args = parser.parse_args()
 
+    if args.from_config:
+        config = load_config_from_toml(args.from_config)
+        cli_config = config.get("optuna_dashboard", {})
+        args.storage = cli_config.get("storage", args.storage)
+        args.storage_class = cli_config.get("storage_class", args.storage_class)
+        args.port = cli_config.get("port", args.port)
+        args.host = cli_config.get("host", args.host)
+        args.server = cli_config.get("server", args.server)
+        args.artifact_dir = cli_config.get("artifact_dir", args.artifact_dir)
+    else:
+        config = {}
+
+    if args.storage is None:
+        raise ValueError("Storage URL must be specified. Use --storage or --from-config.")
```

Comment:
I've noticed that `--storage` does not exist.

```
$ uv run optuna-dashboard --storage sqlite:///db.sqlite3
Traceback (most recent call last):
  File "/Users/c-bata/go/src/github.com/optuna/optuna-dashboard/.venv/bin/optuna-dashboard", line 10, in <module>
    sys.exit(main())
             ^^^^^^
  File "/Users/c-bata/go/src/github.com/optuna/optuna-dashboard/optuna_dashboard/_cli.py", line 142, in main
    raise ValueError("Storage URL must be specified. Use --storage or --from-config.")
ValueError: Storage URL must be specified. Use --storage or --from-config.
```

Could you update the error message like below?

```suggestion
        raise ValueError("Storage URL must be specified as the first argument or via --from-config.")
```

Reply:
You're right. Thank you!